tests/gordo_components/model/test_lstm_autoencoder.py

```diff
@@ -18,8 +19,13 @@ def test_lstm_symmetric_checks_dims(self):
         """
         Test that lstm_symmetric validates parameter requirements
         """
+        # Raise an error if empty dimension and function layers defined.
         with self.assertRaises(ValueError):
-            lstm_symmetric(4, dims=[], funcs=[])
+            lstm_symmetric(4, dims=(), funcs=())
+
```

Comment:
Additionally, you might also guarantee that something like `lstm_symmetric(4)` would work. As mentioned in a previous comment, a bit strange that the default args would illicit a `ValueError` in this function.

Reply:
I ended up landing on comparing the each layer of the different LSTM models' (with deafult parameters) configurations via the `layer.get_config()` method. 

This then ensure that the models pass default config muster and also that they are the same with default config.